.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "protoc": {
+      "options": [
+          "--proto_path=api"
```

Comment:
there are other paths that should be included from the build directory for validate and the google core protos, and protoc itself is also there (`path` option)